iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -413,8 +413,7 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
                 }
```

Comment:
In line 401 we have a default timeout being used. Can we remove that and rely simply on the cancellation token provided?